packages/OntologyService/docs/assets/highlight.css

```diff
@@ -0,0 +1,85 @@
+:root {
```

Comment:
If this is generated - does this need to be included in repo - can you just add it to gh pages?

Reply:
It doesn't no. It could be added to the gh workflow to generate, which should be included when we try to get them working properly

Reply:
Ok. 

Either add the upload to gh pages step
95% sure [this will work](https://github.com/telicent-oss/telicent-ds/blob/main/.github/workflows/upload-storybook.yml) (with minor mods)
OR
Regen on pre-push 

**Rationale**: If we have docs, then they must be forced to be up to date.